build_stats.sh

```diff
@@ -0,0 +1,23 @@
+#! /bin/sh
+
+INST_DIR=$HOME/R-inst
+mkdir -p ${INST_DIR}
+
+# Download a recent version of R (must be 3.3.1 or later).
+wget https://cran.r-project.org/src/base/R-3/R-3.3.2.tar.gz
+tar -xvzf R-3.3.2.tar.gz
+cd R-3.3.2
+
+# Build R.
+./configure --prefix=${INST_DIR} --enable-R-shlib
+make
+make install
+
+cd ..
+
```

Comment:
Ah, I see! Mutating people's `rc` files is a definite no-no. We have two choices.

First, we could add an `echo` to the end saying something like:

```sh
echo "You will need to add $INST_PATH to your $$PATH in order to run bin/warmup_stats"
```

Second -- and perhaps better in this case -- is to simply make `warmup_stats` invoke R from the directory it's been installed into.